vagrant/Vagrantfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+
+    # Ubuntu 16.04 LTS
+    config.vm.box = "fosskers/docker-1.12"
+
+    # Docker permissions are available for `ubuntu` user only.
+    config.ssh.username = "ubuntu"
+
+    # Ports to the services
+    config.vm.network :forwarded_port, guest: 8080, host: 8080
+    config.vm.network :forwarded_port, guest: 9999, host: 9999
+    config.vm.network :forwarded_port, guest: 8777, host: 8777
+
+    # VM resource settings
+    config.vm.provider :virtualbox do |vb|
+        vb.memory = 12288
+        vb.cpus = 4
+    end
+
+    # Provisioning
+    config.vm.provision :shell, path: "bootstrap.sh"
```

Comment:
Having never used Ansible, I wanted to avoid that learning overhead. The provisioning via bash is also fairly lean. 

We had our RRP this morning where I mentioned that Vagrant didn't feel entirely useful for our case here. Between a local `docker-compose` and spinning up a real cluster (quite quickly) with Terraform, Vagrant didn't feel like it had as strong a place in "proof of architecture correctness" as perhaps it does in other projects.